src/bindgen/index.md

```diff
@@ -48,16 +48,17 @@ Then add `bindgen` as a dependency and tell `Cargo` about our build script
 
 `Bindgen` works by reading a `C`-style header file using `libclang`, then
 it runs the preprocessor and creates bindings based on the output. Therefore, 
-before we can start our build script we'll need a dummy header which pulls in
-the `bzip2` library installed on your system.
+the build script needs a dummy header which pulls in the `bzip2` library
+installed on your system.
```

Comment:
Replacing `<bzlib.h>` with `"bzlib.h"` complains about `stdio.h` not being found.
So, I guess it needs to be in the same location as well.
```
./bzlib.h:75:10: fatal error: 'stdio.h' file not found, err: true

--- stderr
./bzlib.h:75:10: fatal error: 'stdio.h' file not found
```